Assets/Code/Interactables/Effects/Static Effects/Model/StaticEffect.cs

```diff
@@ -0,0 +1,40 @@
+﻿public abstract class StaticEffect {
+    // StaticEffects are created BY DynamicEffects and persist across a specific timeframe (a turn, multiple turns, for the rest of the game, etc.)
+    // However, it is possible that a StaticEffect could, in turn, create DynamicEffects as well
+    // For example, a StaticEffect could be something such as a passive of drawing an extra card every turn
+    // The passive would CREATE a DrawCard DynamicEffect every turn
+    // StaticEffects can also be attached to specific entities in the game (status on a character, modifier on a value)
```

Comment:
only AttachedStaticEffects can, no?

Reply:
yes, but StaticEffect is the parent class to both AttachedStaticEffects and PassiveStaticEffects

Reply:
i would put this comment on those classes tho, since it's not technically an attribute of StaticEffects - but that's a nit, i don't really mind having the comment here. just jotting down everything that occurred to me.